scan_engine.h

```diff
@@ -206,7 +206,7 @@ int determineScanGroupSize(int hosts_scanned_so_far,
 
 class UltraScanInfo;
 
-struct ppkt { /* Beginning of ICMP Echo/Timestamp header         */
+struct ppkt { /* Beginning of ICMP Echo/Timestamp header */
```

Comment:
This change isn't necessary for this patch. If you want to make a separate whitespace-only commit to clean up stuff like this, feel free to do so elsewhere.